lib/async.dart

```diff
@@ -2,6 +2,8 @@
 // for details. All rights reserved. Use of this source code is governed by a
 // BSD-style license that can be found in the LICENSE file.
 
+/// {@youtube 560 315 https://www.youtube.com/watch?v=r0tHiCjW2w0}
+
```

Comment:
This is a doc comment, so you should probably add a `library` declaration to make it document the library.

We should also add a real library documentation, but for now, let's just have a headline before the link:
```dart
/// Various functionality associated with the asynchronous features of the `dart:async` library.
///
/// {@youtube 560 315 https://www.youtube.com/watch?v=r0tHiCjW2w0}
library pkg.async;
```

Reply:
> you should probably add a `library` declaration to make it document the library.

Dartdoc should handle it either way - but I'm in favor of the explicit library declaration for now.

I _really_ hope we can make that `library;` soon.

If I can nitpick the wording, I think "Various" and "functionality" aren't conveying much - virtually any package can be describe as having "Various functionality". How about:

```dart
/// Utilities that expand on the asynchronous features of the `dart:async` library.
///
/// {@youtube 560 315 https://www.youtube.com/watch?v=r0tHiCjW2w0}
library pkg.async;
```

Reply:
That would LGTM